src/network/netdev/netdev-gperf.gperf

```diff
@@ -57,6 +58,12 @@ Tunnel.IPv6FlowLabel,        config_parse_ipv6_flowlabel,          0,
 Tunnel.CopyDSCP,             config_parse_bool,                    0,                             offsetof(Tunnel, copy_dscp)
 Tunnel.EncapsulationLimit,   config_parse_encap_limit,             0,                             offsetof(Tunnel, encap_limit)
 Tunnel.Independent,          config_parse_bool,                    0,                             offsetof(Tunnel, independent)
+Tunnel.FOU,                  config_parse_bool,                    0,                             offsetof(Tunnel, fou_tunnel)
+Tunnel.FOUDestinationPort,   config_parse_tunnel_fou_port,         0,                             0
+Tunnel.FOUSourcePort,        config_parse_tunnel_fou_port,         0,                             0
+FouOverUDP.Protocol,         config_parse_uint16,                  0,                             offsetof(FouTunnel, fou_protocol)
```

Comment:
FOU == "Foo Over UDP". What does "FouOverUDP" stand for?

Reply:
Ouch